runtime/src/main/java/org/corfudb/runtime/ViewsGarbageCollector.java

```diff
@@ -75,6 +75,7 @@ public void runRuntimeGC() {
                     currTrimMark);
```

Comment:
getTrimMark is not a safe-call remove Unrecoverable exceptions from its call path.

Reply:
Done.